backend/graphToolsVol2/Estado.py

```diff
@@ -12,3 +12,6 @@ def __init__(self, id, nome):
 
     def visit(self, visitor):
         visitor.visit(self)
+
+    def __lt__(self, other):
```

Comment:
isso é pra `<` só que na lista 28 do arquivo de busca usa `=` não faz sentido... esse acaba não sendo usado mas também não é inútil.